dashboard/pages/api/metric.ts

```diff
@@ -17,17 +17,129 @@
 import { Metrics, MetricsSample } from "../../components/metrics"
 import api from "./api"
 
+export const INTERVAL = 20000
```

Comment:
Will this be too long? Users have to wait for 40 secs+ to get the results.

Reply:
I make a mistake here. Because the promehteus will collect data every 15s-20s. So I though the interval should large than this interval, or the new data will keep the same with old data, which will cause the calculation of BP rate into 0. 

But just realized the data on compute data will update every time.

Will change the interval back to 5s.